### PR TITLE
client for roles of given user and group

### PR DESCRIPTION
**Pull request by zhenz**

### CODE REVIEW

File: src/keycloak/admin/user/usergrouproles.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import json
+
+from keycloak.admin import KeycloakAdminBase
+
+
+class UserGroupRoles(KeycloakAdminBase):
+    _BASE = "/auth/realms/{realm}/api/admin/realms/{realm}/user-group-roles/{user_id}"
```
Comment (diurnalist) [inline src/keycloak/admin/user/usergrouproles.py]: If we're already on a /realms/{realm} path, why is it necessary to filter the realms a second time in the path?

File: src/keycloak/admin/user/usergrouproles.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                )
+            )
+        )
+
+    def grant(self, policy, group_id=None, group_name=None):
+        """
+        grant policy/role to a user for a given group
+        """
+        payload = {
+            "policy": policy
```
Comment (diurnalist) [inline src/keycloak/admin/user/usergrouproles.py:53]: What does "policy" look like? It wasn't totally clear to me from the Keycloak pr. Is it a name of the role?

Comment (zhenz) [reply src/keycloak/admin/user/usergrouproles.py:53]: it's the name of the role, i.e. admin, manager or member

**Review by diurnalist — APPROVED** [review]
LGTM!

**Pull request merged**